backend/src/main/java/de/neuefische/backend/repo/CovidTestRepo.java

```diff
@@ -0,0 +1,17 @@
+package de.neuefische.backend.repo;
+
+import de.neuefische.backend.model.CovidTest;
+import org.springframework.data.repository.PagingAndSortingRepository;
+import org.springframework.stereotype.Repository;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Repository
+public interface CovidTestRepo extends PagingAndSortingRepository<CovidTest, String> {
+
+    List<CovidTest> findAll();
```

Comment:
It should not be necessary to define that method here. A `findAll()` is usually already in the interface you are extending.

Reply:
findAll() returns an Iterable by default, but I want it to return a list